src/lua/api-gateway/util/OauthClient.lua

```diff
@@ -28,6 +39,8 @@ function OauthClient:makeValidateTokenCall(internalPath, oauth_host, oauth_token
         args = { authtoken = oauth_token }
     })
 
+    self:increment(metric)
```

Comment:
can keep this even if the base class has the incrementMetric

Reply:
You should not find any incrementMetric anymore. It's been refactored.